### ISSUE TITLE
Cant figure out how to create a resource pack (actually not an issue but I dont know where else I can ask for help 🤪)

### ISSUE THREAD

**Issue by Waillio** [sisby-folk/antique-atlas-respacks#7]
Hello!
Cant find out a proper way to reach to developers so thought about creating an issue here.

I'm using different gen mods in my modpack for myself and my friends and thought about creating resource pack to increase compatibility with them. Before actually making textures I tried to link BOP nether biomes to AA4 following the guide and failed miserably few times and dont know what I'm doing wrong. Can anyone please direct me a bit?

So I created a path for one of a biomes for BOP according to instructions, like this (naming are correct both for mod name and biome ID) -

assets\biomesoplenty\atlas\biome\undergrowth.json

And for testing purposes linked it to existing textures, like this -

{
  "textures": {
    "antique_atlas:biome/woodland/forest/oak_fancy_mixed": 1,
    "antique_atlas:biome/woodland/forest/oak_mixed": 1,
    "antique_atlas:biome/woodland/forest/grass_oak_mixed": 1
  }
}

Putted it into minecraft (trough resource packs) etc and nothing happens. I'm definitely doing something incredibly wrong but cant figure out what exactly - since I have zero experience in creating RP or mods (only translating experience). Any chance somebody can point me to my mistakes? Thanks in advance!

**Comment by sleepingdragoninn** [comment]
can you send the full resource pack zip you made?

**Comment by Waillio** [comment]
Sure
[BOP_AA4_compat.zip](https://github.com/user-attachments/files/17404597/BOP_AA4_compat.zip)

**Comment by sleepingdragoninn** [comment]
looks fine first glance, can you turn on debug mode via antique_atlas.toml and hover the mouse over that biome on the atlas and take a screenshot?

**Comment by Waillio** [comment]
Okay, I tested it and it looks like it actually works but not always? It works somewhere, but somewhere its not. I suppose its just nether thing or changed worldgen? I regenerated (I suppose I did it right) map by deleting "surveyor" in a save folder, but it still looks like that. 
![image](https://github.com/user-attachments/assets/f2eeb26e-3b8a-44e0-9c90-d69ff82a05b9)
![image](https://github.com/user-attachments/assets/9e52ddfc-0490-46d0-9282-103a24b11ccf)
![image](https://github.com/user-attachments/assets/bc53e9ba-16b8-46b9-ad03-a3e685ee3bda)
